### ISSUE TITLE
Blade Trap is missing mapping+causes crash upon world creation

### ISSUE THREAD

**Issue by Nat-cha034** [Sunconure11/dungeon-mobs#45]
Just downloaded the newest version of the mod last night, & upon attempting to load a new world with it, MC froze upon world creation & then crashed, according to the crash log, this is due to the Blade Trap missing a mapping:

https://paste.dimdev.org/xudakoxihu.rb

**Comment by BlackJar72** [comment]
That's very strange, since blade traps were not changed!  I'll see if I can take a look next week; sorry, can't do it today.

**Comment by BlackJar72** [comment]
The code does not seem to have changed from the previous version.

**Comment by Ariannae** [comment]
It looks like I'm also able to corroborate this, as my Minecraft is crashing related to Blade Traps with the exact same prompt as Nat-cha034.

In addition, it looks like the log gets spammed with this prior to crashing:
```
[17:06:48] [Server thread/ERROR] [FML]: A TileEntity type com.gw.dm.blocks.TileEntityBladeTrap has throw an exception trying to write state. It will not persist. Report this to the mod author
java.lang.RuntimeException: class com.gw.dm.blocks.TileEntityBladeTrap is missing a mapping! This is a bug!
	at net.minecraft.tileentity.TileEntity.writeInternal(TileEntity.java:78) ~[avj.class:?]
	at net.minecraft.tileentity.TileEntity.writeToNBT(TileEntity.java:69) ~[avj.class:?]
	at com.gw.dm.blocks.TileEntityBladeTrap.writeToNBT(TileEntityBladeTrap.java:192) ~[TileEntityBladeTrap.class:?]
	at net.minecraft.world.chunk.storage.AnvilChunkLoader.writeChunkToNBT(AnvilChunkLoader.java:383) [aye.class:?]
	at net.minecraft.world.chunk.storage.AnvilChunkLoader.saveChunk(AnvilChunkLoader.java:174) [aye.class:?]
	at net.minecraft.world.gen.ChunkProviderServer.saveChunkData(ChunkProviderServer.java:202) [on.class:?]
	at net.minecraft.world.gen.ChunkProviderServer.saveChunks(ChunkProviderServer.java:230) [on.class:?]
	at net.minecraft.world.WorldServer.saveAllChunks(WorldServer.java:1007) [oo.class:?]
	at net.minecraft.server.MinecraftServer.saveAllWorlds(MinecraftServer.java:408) [MinecraftServer.class:?]
	at net.minecraft.server.integrated.IntegratedServer.saveAllWorlds(IntegratedServer.java:376) [chd.class:?]
	at net.minecraft.server.MinecraftServer.stopServer(MinecraftServer.java:446) [MinecraftServer.class:?]
	at net.minecraft.server.integrated.IntegratedServer.stopServer(IntegratedServer.java:515) [chd.class:?]
	at net.minecraft.server.MinecraftServer.run(MinecraftServer.java:579) [MinecraftServer.class:?]
	at java.lang.Thread.run(Unknown Source) [?:1.8.0_271]
```

**Comment by BlackJar72** [comment]
I'll see when I can get to this.

**Comment by BlackJar72** [comment]
OK, it looks like it just need the mod id in the event subscription to work; it should be fixed now (well, as soon as I compile and upload, I may check to make sure the fixes in the last patch are there in what I currently have).

**Comment by BlackJar72** [comment]
OK, the problem seems to be fixed in the new patch, version 5.8.8.
